velox/core/PlanNode.h

```diff
@@ -15,6 +15,8 @@
  */
 #pragma once
 
+#include <folly/container/F14Set.h>
```

Comment:
Not needed?

Reply:
Moved to `PlanNode.cpp`, it's required by `folly::F14FastSet<std::string> sortingKeyNames`.
Thanks!